packages/stream-model-instance/src/model-instance-document.ts

```diff
@@ -32,30 +32,6 @@ export interface ModelInstanceDocumentMetadataArgs {
    */
   controllers?: Array<string>
 
-  /**
-   * Allows grouping similar documents into "families". Primarily used by indexing services.
-   */
-  family?: string
-
-  /**
-   * Allows tagging documents with additional information. Primarily used by indexing services.
-   */
-  tags?: Array<string>
-
-  /**
-   * If specified, must refer to another ModelInstanceDocument whose contents are a JSON-schema specification.
-   * The content of this document will then be enforced to conform to the linked schema.
-   */
-  schema?: CommitID | string
-
-  /**
-   * If true, then two calls to ModelInstanceDocument.create() with the same content and the same metadata
-   * will only create a single document with the same StreamID. If false, then otherwise
-   * identical documents will generate unique StreamIDs and be able to be updated independently.
-   * @deprecated use deterministic function instead
-   */
-  deterministic?: boolean
-
   /**
    * If true, all changes to the 'controllers' array are disallowed.  This guarantees that the
    * Stream will always have the same controller. Especially useful for Streams controlled by
```

Comment:
The `forbidControllerChange` should be removed imo. It should never be possible to change controller of a MID.

Reply:
done.